webpackBaseConfig.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
   resolve: {
     modules: [__dirname, 'node_modules'],
     alias: {
-      '@mui/docs': path.resolve(__dirname, './node_modules/@mui/monorepo/packages/mui-docs/src'),
```

Comment:
This did not seem to be used anywhere. 🤔 🤷

Reply:
i believe this is used by eslint

Reply:
> i believe this is used by eslint

How can we test if it is needed given that no `ESLint` issues are triggered by removing it? 🤔

Reply:
I reverted this change for now as it seems that we critically need this change or removing the explicit `@mui/docs` package altogether to fix this issue:

https://github.com/mui/mui-x/assets/4941090/7b4c0345-c79a-406e-a64a-80c81d48c66b

Reply:
> How can we test if it is needed given that no ESLint issues are triggered by removing it? 🤔

technically it is needed when relying on aliasing this package from the monorepo. it's working when you remove the rule because the package is also installed and there is a high chance it's compatible with the aliased version. remove it from package.json and I suspect eslint will break without this.

Reply:
Gotcha. Thanks for the clarification.
I've left the alias intact. 👌